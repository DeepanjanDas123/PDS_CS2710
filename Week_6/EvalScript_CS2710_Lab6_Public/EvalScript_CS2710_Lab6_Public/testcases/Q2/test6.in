11
NEW PUSH_BEGIN 5 PUSH_END 15 PUSH_BEGIN 10 PUSH_END 20 PUSH_BEGIN 25 DELETE 10 REVERSE 3 ROTATE 2 PUSH_END 30 DELETE 20