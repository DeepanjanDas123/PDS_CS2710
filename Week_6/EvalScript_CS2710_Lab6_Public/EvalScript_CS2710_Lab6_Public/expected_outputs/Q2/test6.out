25 15 5 30 